src/views/data.ts

```diff
@@ -389,6 +391,93 @@ const data = [
       
       }`,
             },
+            {
+                route: "GET /address?city=Mumbai",
+                content: "Returns dummy address from perticular city.",
+                output: `[
+
+                {
+                    "street": "72,Pragee shopping centre,bajaj road",
+                    "city": "Mumbai",
+                    "state": "Maharastra",
+                    "country": "India",
+                    "pincode": "400056"
+                },
+                {
+                    "street": "A/7, New Sonal Industrial Estate",
+                    "city": "Mumbai",
+                    "state": "Maharastra",
+                    "country": "India",
+                    "pincode": "400072"
+                },
+                {
+                    "street": "Ganesh Nagar,charkop kandivali (west)",
+                    "city": "Mumbai",
+                    "state": "Maharastra",
+                    "country": "India",
+                    "pincode": "400067"
+                }
+            
+            ]`,
+            },
+            {
+                route: "GET /address?state=Maharastra",
+                content: "Returns dummy address from perticular state.",
+                output: `[
+
+                  {
+                      "street": "72,Pragee shopping centre,bajaj road",
+                      "city": "Mumbai",
+                      "state": "Maharastra",
+                      "country": "India",
+                      "pincode": "400056"
+                  },
+                  {
+                      "street": "A/7, New Sonal Industrial Estate",
+                      "city": "Mumbai",
+                      "state": "Maharastra",
+                      "country": "India",
+                      "pincode": "400072"
+                  },
+                  {
+                      "street": "Ganesh Nagar,charkop kandivali (west)",
+                      "city": "Mumbai",
+                      "state": "Maharastra",
+                      "country": "India",
+                      "pincode": "400067"
+                  }
+              
+              ]`,
+            },
+            {
+                route: "GET /address?city=Mumbai&state=Maharastra&country=India",
+                content: "Return dummy data after chaining multiple address query parameter.",
+                output: `[
+
```

Comment:
Add a route that queries only two fields. Say `city` and `state`

Reply:
Done! Plz review